tests/integration/server.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+from google.cloud import logging as gcloud_logging
+from google.cloud import monitoring as gcloud_monitoring
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.exceptions import Forbidden as ForbiddenException
+from google.cloud.exceptions import NotFound as NotFoundException
+from oauth2client.client import GoogleCredentials
+
+import logging
```

Comment:
standard library imports go in the first section, third-party imports in the second.